iommi/edit_table__tests.py

```diff
@@ -1,6 +1,10 @@
 import pytest
 
-from docs.models import Album
+from docs.models import (
+    Album,
+    Artist,
+    Profile,
+)
```

Comment:
Was there any test here that didn't get included?

Reply:
Yep, I moved it to docs. Fixed.